app/js/components/App.jsx

```diff
@@ -0,0 +1,21 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+import React from 'react';
+
+import LabResultslist from '../components/LabResultslist/LabResultslist';
```

Comment:
Looks like you took this directly from the ticket, but thinking about this further the ticket probably was poorly named... this is a list of Lab Orders, not Lab Results, so I would change this component name to LabOrdersList